docs/src/index.md

```diff
@@ -0,0 +1,118 @@
+# GeoMakie.jl
+GeoMakie.jl is a Julia package for plotting geospatial data on a given map projection. It is based on the [Makie.jl plotting ecosystem](https://makie.juliaplots.org/stable/).
+
+The package [ClimateBase.jl](https://juliaclimate.github.io/ClimateBase.jl/dev/) builds upon GeoMakie.jl to create a seamless workflow between analyzing/manipulating climate data, and plotting them.
+
+## Installation
+
+This package is **in development** and may **break**, although we are currently working on a long-term stable interface. 
+
+You can install it from the REPL like so:
+```julia
+]add GeoMakie
+```
+
+## API
```

Comment:
```suggestion
## GeoAxis
```

Reply:
I disagree with this proposed change.

Reply:
GeoAxis will be the new member of the Makie family, i.e. `Axis` and `Axis3`, and here you introduce how to do things with GeoMakie, hence it seems appropriate to call this sections GeoAxis and API in my opinion doesn't convey or better said introduces properly the functionality.